pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -137,7 +138,10 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 		return fmt.Errorf("couldn't create deployment for deployment config %s: %v", deployutil.LabelForDeploymentConfig(config), err)
 	}
 	c.recorder.Eventf(config, "DeploymentCreated", "Created new deployment %q for version %d", created.Name, config.LatestVersion)
-	return nil
+	// Update deploymentConfig status; we probably need a separate call for status
```

Comment:
If we're introducing the idea of a `status`, let's introduce a `status` at least internally.  Since `status` should be read-only, we can be compatible by duplicating existing fields in two locations.

Does the deployer pod update logical status?  If so, then we'll need to need to respect changes on it if the `status` stanza is missing.  If not and its only the baked in controller, then we should be good to ignore status updates on old fields.